### ISSUE TITLE
publish to JSR

### ISSUE THREAD

**Issue by birkskyum** [maplibre/maplibre-gl-js#3804]
It would be great to get on the [JSR registry](https://jsr.io/) as it's taking off.

Since it's esm-only this is blocked by:
- #1595

**Comment by HarelM** [comment]
Thanks for sharing!
I think we might be able to try again and export an esm build now that I have removed the global object.
If it's only adding another entry in the rollup config it should be failry easily.
I don't know about the other part related to JSR and I'll be surprised if we could upload the code "as is", maybe the csp code can be uploaded as it's "less complicated" I believe, IDK, would be interesting to try out.

**Comment by birkskyum** [comment]
I think it'll be significantly harder to land jsr support than manually making the esm bundles (#1595), because jsr.io wan't to take .ts source code and make a build, .d.ts, docs, etc., and I don't know how it'll will handle our need for multiple bundles - if it reads our rollup.config.

**Comment by HarelM** [comment]
Yeah, although if we still need a rollup.config file and all we get is d.ts file and API docs (and if I needed to guess they probably are using the same tools we are using - dts bundle generator or api-extractor, and typedoc) then there's not a lot of added value here.
But it probably can't hurt to keep an eye on this.

**Comment by louwers** [comment]
Sent an invite to @birkskyum for the `@maplibre` scope on JSR.
